### ISSUE TITLE
Cannot open it in unity

### ISSUE THREAD

**Issue by raulseixasbr** [shreyyasm/KillStreak#1]
Hi! I was impressed with the game, and tried to open it in unity. But It says all the scenes couldn't be opened because they are corrupted. Are some needed files missing?

Thank you

**Comment by shreyyasm** [comment]
Yeah Beacause of the project packages dependencies! I will add a readme for
that So you can add those packages by given links soon.

**Comment by raulseixasbr** [comment]
Looking forward to that. Thank you!
